deep_water_level/src/infer.py

```diff
@@ -231,13 +245,18 @@ def on_add(sel):
         "--use_water_line",
         type=bool,
         default=False,
-        help="If set, do inference of the water level coordinates as well as depth",
+        help="If set, do inference of the water level coordinates as output instead of depth",
     )
 
-    parser.add_argument("--verbose", type=bool, default=False, help="Print error for each image in the dataset")
+    parser.add_argument(
```

Comment:
I think one of my PRs switched this to 1 line.  Maybe we have something configured differently?

in ruff.toml I have 'line-length = 120', do you have the same?

Reply:
I changed this line and made the argument `store_true` so that you can call `infer.py --verbose` instead of needing `infer.py --verbose=1`, so i don't think this is a case of the formatter being random. Yeah i also have `line-length=120`